### ISSUE TITLE
Translation files require an update

### ISSUE THREAD

**Issue by arcctgx** [lanoxx/tilda#508]
Most of the `.po` files seem to be outdated. In my testing I updated the `POTFILES.in` file to make sure all translatable strings are included. Once it's done, updating translation files automatically by `make update-po` results in a massive change set:

```
$ git diff --numstat
(...)

213     165     po/bg.po
213     163     po/ca.po
213     165     po/cs.po
217     163     po/de.po
211     162     po/el.po
214     166     po/en_GB.po
220     170     po/es.po
212     164     po/fr.po
216     162     po/hr.po
213     165     po/hu.po
214     163     po/it.po
217     166     po/lt.po
216     162     po/nb.po
211     164     po/pl.po
213     165     po/pt.po
218     164     po/pt_BR.po
216     165     po/ru.po
214     166     po/sk.po
213     163     po/sl.po
214     166     po/sv.po
211     162     po/tr.po
83      64      po/uk.po
216     182     po/zh_CN.po
217     183     po/zh_TW.po
```

Or, in summary:
```
24 files changed, 5015 insertions(+), 3880 deletions(-)
```

Full set of changes can be examined here: https://github.com/lanoxx/tilda/compare/master...arcctgx:tilda:update-po. As expected, most of these changes follow changes in line numbers. But after examining the Polish translation I can also see that some strings become assigned incorrectly. If that's happening in one translation, it's probably affecting others as well.

I'm not confident that such "big bang" update is the right thing to do, but how should we proceed? I can fix mistakes in the Polish translation, but what about other languages?

**Comment by lanoxx** [comment]
In the past I did run `make update-po` from time to time in the `build/po` directory and then commit the results into the repo. As far as I can tell, Gettext is smart enough to update line number changes and it seems it can also detect when a string moves to a new place and update the po files accordingly.

For example here we moves some strings from menu.ui to tilda-context-menu.c and this is correctly reflected after running `update-po`:

```
-#: src/menu.ui:6
-msgid "_New Tab"
-msgstr "_Нов подпрозорец"
```

+#: src/tilda-context-menu.c:174
+msgid "_New Tab"
+msgstr "_Нов подпрозорец"

As I don't have the capacity to validate all translations, I guess we just need to wait and see if we get some merge requests from the community.

For now I pushed a new translation update in this commit: 3f81b0b1b94d310128ee9b5dd5d688e9683b28fb
